oe_content.post_update.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * @file
+ * Post update functions for OpenEuropa Content module.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\media\Entity\MediaType;
+
+/**
+ * Remove the mapping of name to title on documents and image media types.
+ */
+function oe_content_post_update_0001(): void {
```

Comment:
as I remember we agreed to have 5 digits.
```suggestion
function oe_content_post_update_00001(): void {
```